contrib/backports/backports-update.md

```diff
@@ -0,0 +1,72 @@
+Bitcoin Backports
+====================
+Bitcoin Backports are an incredibly valuable part of Dash's development. Backporting allows us to easily implement new
+features, improvements and fixes as bitcoin implements them.
+
+To see detailed statistics & progress see Google Sheet tracker: [Bitcoin backports for Dash](https://docs.google.com/spreadsheets/d/1DnKxat0S0H62CJOzXpKGPXTa8hgoVOjGYZzoClmGSB8/edit?usp=sharing).
+You should use this sheet to find what PRs to backport and its commit. 
+
+Updating the Spreadsheet
+-------------------
+To keep the spreadsheet up to date we need to pull merges made to Bitcoin for each version.
+
+### Adding Bitcoin Remote
+
+If you have not already, add the bitcoin repo as a remote:
+
+```
+git remote add bitcoin git@github.com:bitcoin/bitcoin.git && git fetch bitcoin
+```
+
+This allows you to easily cherry-pick merges and look into logs of bitcoin without switching directories.
+
+### Pulling Merges
+
+To pull the most up-to-date merges first make sure bitcoin is up-to-date:
+
+```
+git fetch bitcoin
```

Comment:
nit, you already fetched bitcoin, could remove the one above

Reply:
I left that in for the purpose of making sure that master is up to date if someone already has the remote added but not updated